colossus-docs/src/main/scala/StreamingHttpExample.scala

```diff
@@ -24,6 +27,12 @@ object StreamServiceExample {
           )
         }
       }
+
+      case _ => Callback.successful(StreamingHttpResponse(

```

Comment:
What's this about?

Reply:
When the example run, if only one route is defined, and other is hit nothing happens, this way we can return a 404 and give feedback when interacting with the example

Reply:
Sounds like a bug? Can you create a ticket if so.